modules/kmm-setting-soft-dependencies-between-kernel-modules.adoc

```diff
@@ -0,0 +1,41 @@
+// Module included in the following assemblies:
+//
+// * hardware_enablement/kmm-kernel-module-management.adoc
+
+:_content-type: CONCEPT
+[id="kmm-setting-soft-dependencies-between-kernel-modules_{context}"]
+= Set soft dependencies between kernel modules
+
+Some configurations require that several kernel modules be loaded in a specific order to work properly, even though the modules do not directly depend on each other through symbols.
+These are called soft dependencies.
+`depmod` is usually not aware of these dependencies, and they do not appear in the files it produces.
+For example, if `mod_a` has a soft dependency on `mod_b`, `modprobe mod_a` will not load `mod_b`.
```

Comment:
Will users likely know what depmod and modprobe are?

Reply:
@mburke5678 Yes.